df_chat/models.py

```diff
@@ -89,13 +89,28 @@ def get_upload_to(self, filename):
     image = models.ImageField(upload_to=get_upload_to, null=True, blank=True)
 
     is_public = models.BooleanField(default=True)
+    
+    # ! This is unnecessary when there exists a through-model `RoomUser` model.
     users = models.ManyToManyField(User, blank=True)
+    
     admins = models.ManyToManyField(User, blank=True, related_name="rooms_admin_set")
 
     muted_by = models.ManyToManyField(User, blank=True, related_name="room_muted_set")
 
     objects = RoomQuerySet.as_manager()
 
+    @property
+    def users_count(self) -> int:
```

Comment:
Why do we need this method? 

`room.users.count()` seems self-explanatory to me

Reply:
Done!